chart/portworx/values.yaml

```diff
@@ -55,3 +55,4 @@ serviceAccount:
 deleteStrategy:                      # Delete strategy for the portworx cluster
   type:                              # Valid values: Uninstall, UninstallAndWipe
 
+kubeletDir: /var/lib/kubelet         # custom kubelet dir
```

Comment:
@Glitchfix change this to `/var/data/kubelet` . Also check what is the kubelet path on IBM OCP clusters.

Reply:
it uses the same `/var/data/kubelet`
```sh
sh-4.4# ls -la /var/lib/kubelet 
lrwxrwxrwx. 1 root root 17 Dec  1 05:43 /var/lib/kubelet -> /var/data/kubelet
```